src/components/reusable_components/CustomSearch.js

```diff
@@ -69,43 +40,39 @@ export default function CustomSearch(props) {
         }}
       />
       {/* Render the list of results after loading is done */}
-      {searchResults && !isLoading ? (
+      {searchResults && !isLoading && searchQuery !== "" ? (
         <FlashList
           data={searchResults}
-          keyExtractor={(item) => {
-            return item.id;
-          }}
+          keyExtractor={(item, index) => index.toString()}
           viewabilityConfig={{
             waitForInteraction: true,
             itemVisiblePercentThreshold: 100,
             minimumViewTime: 1000,
           }}
-          estimatedItemSize={mockData.length}
-          onEndReached={() => console.log("Reached bottom")}
+          estimatedItemSize={searchResults.length}
           renderItem={({ item }) => (
             <TouchableOpacity
               style={styles.resultListItem}
               onPress={() =>
-                searchType === "standard"
+                searchType === "standard" &&
+                !searchResults.includes("Ticker not found")
```

Comment:
The navigating shouldn't be done here, this stil makes it specific to the StockInfoScreen.

Reply:
done